model/jpa/src/main/resources/META-INF/jpa-changelog-1.0.0.Final.xml

```diff
@@ -18,7 +18,10 @@
 
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.2.xsd">
     <changeSet author="sthorger@redhat.com" id="1.0.0.Final">
```

Comment:
The changeset IDs should be modified (e.g. to 1.0.0.Final-KEYCLOAK-5461) to avoid checksum changes. This applies to all affected changesets.